src/mo_types/type.mli

```diff
@@ -133,6 +134,7 @@ val is_unit : typ -> bool
 val is_pair : typ -> bool
 val is_func : typ -> bool
 val is_async : typ -> bool
+val is_low_async_fut : typ -> bool
```

Comment:
Is this code dead now? I can only find the definition not any use. Deletre?

Reply:
I believe I removed this 😬

Reply:
Yeah, dead, will zap!

Reply:
8b98dc928